include/jemalloc/internal/tsd_malloc_thread_cleanup.h

```diff
@@ -3,6 +3,9 @@
 #endif
 #define JEMALLOC_INTERNAL_TSD_MALLOC_THREAD_CLEANUP_H
 
+#include "jemalloc/internal/tsd_internals.h"
+
+#ifdef JEMALLOC_MALLOC_THREAD_CLEANUP
```

Comment:
This shouldn't be necessary? https://github.com/jemalloc/jemalloc/blob/36ca0c1b7de5fc92e6be48f73f28a6dce0e8890e/include/jemalloc/internal/tsd.h#L307

Reply:
Good point. I think this was a misguided attempt to make the header compile before I realized that `tsd_malloc_thread_cleanup.h` isn't used on Linux, so of course it shouldn't compile (in a similar fashion to how even after this PR `tsd_win.h` does not compile on Linux, because of course it doesn't, it's for Windows).